src/app/projectmanagement/overview.component.ts

```diff
@@ -1044,7 +1044,7 @@ export class OverviewComponent extends ApplicationBaseClassComponent implements
 		if (this.project_application.project_application_pi.elixir_id === this.userinfo.ElixirId) {
 			this.updateNotificationModal(
 				'Denied',
-				'You cannot leave projects as PI. Please contact cloud@denbi.de for further steps.',
+				'You cannot leave projects as PI. Please contact cloud-helpdesk@denbi.de for further steps.',
```

Comment:
Should be changed to use  CLOUD_PORTAL_SUPPORT_MAIL (from links.ts) so if mail is changed again we do not need to change it anywhere else